template-roles-and-resources.md

```diff
@@ -0,0 +1,167 @@
+- Felicity Brand
+- Your name here! There is no limit to how many can join this group!
+
+
+### Responsibilities
+
+Community reviewers:
+- Respond to requests from the templates coordinator and/or template mentors when a community member has a draft that needs review.
+- Provide meaningful feedback that will help the template produce a high-quality template.
+- Encourage both global-level revisions (on the document as a whole) and local-level revisions (at the paragraph, sentence, or word level) as needed.
+- Strive to give feedback that is constructive and avoid making comments that will increase defensiveness.
+- Be available (asynchronously) to the templateer if they have follow-up questions or concerns about your feedback.
+
+
+### Eligibility
+
+To become a community reviewer:
+- You should be a member in good standing in the Good Docs Project community, as defined in our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/).
+- You should have writing experience or subject matter expertise for the type of document being reviewed.
+- You should be familiar with the [Conventional Comments](https://conventionalcomments.org/) theory of labeling feedback.
+- Let the current templates coordinator or a template mentor know that you want to volunteer as a community reviewer.
+
+
+## Pull request reviewers
+
+This group of experienced templateers review and approve pull requests to the templates repository.
+
+> :triangular_flag_on_post: **NOTE: At least two pull request reviewer approvals are necessary to merge a template into the project.**
+
+The current pull request reviewers are:
+
+- Aidan Doherty
+- Alyssa Rock
+- Cameron Shorter
+
+### Responsibilities
+
+Pull request reviewers:
+- Act as gatekeepers for template quality and assist in making the final judgment call about when a template is ready to merge.
+- Apply the [template quality checklist](template-quality-checklist.md) to each pull request to ensure it meets the project’s standards for quality.
+- Strive to give feedback that is constructive and avoid making comments that will increase defensiveness.
+- Be available (asynchronously) to the templateer if they have follow-up questions or concerns about your feedback.
+- Coordinate and communicate regularly with the templates coordinator.
+- Work with the templates coordinator to ensure each template receives at least two reviews from pull request reviewers when a template is in the pull request phase.
+- Notify the templates coordinator and make recommendations about when a template is ready to be merged.
+
+
+### Eligibility
+
+To become a pull request reviewer:
+- You should be a member in good standing in the Good Docs Project community, as defined in our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/).
+- You should be familiar with the Conventional Comments theory of labeling feedback.
+- You should be an experienced technical writer who is familiar with our project’s standards for quality.

```

Comment:
thought: Experienced technical writer is an interesting term to consider. I would consider this no barrier to my participation. But compared to someone like Cameron, I might not be considered very experienced. I hope "impostor syndrome" or just not thinking of oneself as experienced isn't too much of a barrier for people to step up.

Reply:
You're going to be just fine, Aidan. :)